pymunk/shapes.py

```diff
@@ -578,14 +578,16 @@ def __init__(
     ) -> None:
         """Create a polygon.
 
-        A convex hull will be calculated from the vertexes automatically.
+        A convex hull will be calculated from the vertexes automatically. Note
+        that concave ones will be converted to a convex hull using the Quickhull
+        algorithm.
 
-        Adding a small radius will bevel the corners and can significantly
+        Add a small radius will bevel the corners and can significantly
```

Comment:
I think Adding is the correct way here. This is what the user would need pass in to this method, not what the method does.